src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Functions/Processor.cs

```diff
@@ -15,59 +12,60 @@ namespace GovUk.Education.ExploreEducationStatistics.Data.Processor.Functions
     // ReSharper disable once UnusedMember.Global
     public class Processor
     {
-        private readonly IFileImportService _fileImportService;
         private readonly IDataImportService _dataImportService;
         private readonly IProcessorService _processorService;
-        private readonly IDbContextSupplier _dbContextSupplier;
         private readonly ILogger<Processor> _logger;
 
         public Processor(
-            IFileImportService fileImportService,
             IDataImportService dataImportService,
             IProcessorService processorService,
-            IDbContextSupplier dbContextSupplier,
             ILogger<Processor> logger)
         {
-            _fileImportService = fileImportService;
             _dataImportService = dataImportService;
             _processorService = processorService;
-            _dbContextSupplier = dbContextSupplier;
             _logger = logger;
         }
 
         [FunctionName("ProcessUploads")]
         public async Task ProcessUploads(
             [QueueTrigger(ImportsPendingQueue)] ImportMessage message,
             ExecutionContext executionContext,
-            [Queue(ImportsPendingQueue)] ICollector<ImportMessage> importStagesMessageQueue,
-            [Queue(ImportsAvailableQueue)] ICollector<ImportObservationsMessage> importObservationsMessageQueue
```

Comment:
I think the `ImportsAvailableQueue` is now removed? If so we could have a faffy deploy ticket on removing it along with `ImportsAvailablePoisonQueue` from all environments, and remove both of them from `ImporterQueues`.

We can also remove the `ImportObservationsPoisonHandler` function.